install.ps1

```diff
@@ -877,18 +734,192 @@ if (-not $noGui.IsPresent) {
         $newXmlNode.SetAttribute("name", "RAW_TOOLS_DIR")
         $newXmlNode.SetAttribute("value", $rawToolsDirText.text)
 
-        # Add selected packages
-        $packages = $configXml.SelectSingleNode('//packages')
-        foreach($package in $selectedPackagesBox.Items) {
-            $newXmlNode = $packages.AppendChild($configXml.CreateElement("package"))
-            $newXmlNode.SetAttribute("name", $package)
-        }
+
+        [void]$formEnv.Close()
+
     } else {
         Write-Host "[+] Cancel pressed, stopping installation..."
         Start-Sleep 3
         exit 1
     }
 
+    ################################################################################
+    ## PACKAGE SELECTION BY CATEGORY
+    ################################################################################
+
+    function Install-Selected-Packages
```

Comment:
Please add a short comment to document what the function does in all the added functions.